.github/workflows/build-library.yml

```diff
@@ -71,14 +71,8 @@ jobs:
         with:
           node-version: 13.3.0
           registry-url: https://registry.npmjs.org/
-      - run: npm install
-      - name: Publish if version has been updated
-        uses: pascalgn/npm-publish-action@1.1.0
-        with:
-          tag_name: "v%s"
-          tag_message: "v%s"
-          commit_pattern: "^Release (\\S+)"
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          NPM_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - run: npm ci
+      - run: npm build
+      - name: Try to publish to npm
+        run: npm publish --access public
         continue-on-error: true
```

Comment:
Why do we need this?

Reply:
We are not changing the version every time, sometimes we do a PR just for updating README, and that will result in failure of publishing, then we will see a cross with the merged commit message.

Reply:
Then the action shouldn't error, I think. Can we change that?

Reply:
Or maybe only run the publish step if the version has changed.